Documentation/dev/libvirt-howto.md

```diff
@@ -40,7 +40,50 @@ polkit.addRule(function(action, subject) {
 EOF
 ```
 
-#### 1.6 Prepare the configuration file
+#### 1.6 Configure libvirt to accept TCP connections
+
+The Kubernetes [cluster-api](https://github.com/kubernetes-sigs/cluster-api)
+components drive deployment of worker machines.  The libvirt cluster-api
+provider will run inside the local cluster, and will need to connect back to
+the libvirt instance on the host machine to deploy workers.
+
+In order for this to work, you'll need to enable TCP connections for libvirt.
+To do this, first modify your `/etc/libvirt/libvirtd.conf` and set the
+following:
+```
+listen_tls = 0
+listen_tcp = 1
+auth_tcp="none"
+tcp_port = "16509"
+```
+
+Note that authentication is not currently supported, but should be soon.
```

Comment:
Are there blockers to supporting TLS at least?  Or a way to restrict the listening to an interface that's only exposed to libvirt domains?  Relying on the firewall as the only thing between external users and control of my libvirtd makes me jumpy.  But I'm not very familiar with libvirt, so maybe I'm just misunderstanding :p.

Reply:
I'm working on TLS authentication support in the actuator now. It's not there yet, but will be soon.